scss/civicrm/administer/events/_badgelayout.scss

```diff
@@ -0,0 +1,23 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+&#{civi-page('admin-badgelayout')} {
+  .crm-container {
+    h3 {
```

Comment:
i don't get it, isn't this supposed to happen automatically at this point?

Reply:
well can't we have a global rule then for both of these positions?

Reply:
Actually no, if we have title inside block but on first first postion then we style it as second screenshot, just we won't need shadow in that case. and generally it's shared by same containers so. there is not way to identify it separately.